analysis/src/topObjectSelection.cc

```diff
@@ -98,8 +98,15 @@ vector<TParticle> topObjectSelection::vetoMuonSelection() {
 }
 
 vector<TParticle> topObjectSelection::jetSelection() {
+  BTagCalibration calib("csvv2", "/cms/scratch/jdj0715/nanoAOD/src/nano/analysis/data/btagSF/CSVv2_Moriond17_B_H.csv");
```

Comment:
Theres already a btagcalibration object in nanoBase that picks up the csv file based on the CMSSW environment, use that instead of creating your own object